src/components/Contact/ContactBand/ContactBand.js

```diff
@@ -0,0 +1,47 @@
+import { Button, Container, Grid, Paper, Typography } from '@material-ui/core';
+import React from 'react';
+import injectSheet from 'react-jss';
+import styles from './ContactBand.styles';
+import { MAIN_IMAGES } from '../../../../data/templateMeta';
+
+const ContactBand = ({ classes }) => {
+    const resumeLink = '../../../../data/resume';
+
+    return (
+        <Container component='section' className={classes.root}>
+            <Typography variant='h3' component='h1' color='primary' gutterBottom>Get in touch</Typography>
+            <Typography variant='h6' component='h2' color='primary' gutterBottom>This is how you can reach me.</Typography>
+            <hr class={classes.contactRefWrapper} />
+            <Grid className={classes.contactRefWrapper} container spacing={6}>
+                <Grid item xs={12} sm={6}>
+                    <Paper className={classes.paper}>
+                        <Typography align='center'>
+                            <Typography component='span' color='secondary'>Email: </Typography>
+                            wells.sn2@gmail.com
```

Comment:
Note: I plan to consolidate text across components into their own data files in a separate pr